### PR TITLE
Avoid going through the Redux store when adding a credit card

### PR DESCRIPTION
**Pull request by cloudify**

### CODE REVIEW

File: ts/sagas/wallet.ts [hunk 1]
```diff
@@ -863,7 +854,8 @@ export function* watchWalletSaga(pagoPaClient: PagoPaClient): Iterator<Effect> {
     if (action.type === ADD_CREDIT_CARD_REQUEST) {
       yield fork(
```
Comment (cloudify) [inline ts/sagas/wallet.ts]: @fgiobergia btw why is this a fork instead of a call?

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cloudify** [comment]
Yes indeed, but what other request you would handle before handling the result of saving a card? I believe the app will actually wait for the card to be saved to present a feedback to the user?

Inviato da iPhone

**Comment by cloudify** [comment]
I mean, by forking that saga, the user could save another card and while he's doing that he could receive an errore be cose the first card he was trying to save has failed.

Inviato da iPhone

File: ts/sagas/wallet.ts (hunk 1 above)
Comment (cloudify) [reply ts/sagas/wallet.ts]: @fgiobergia Yes indeed, but what other request you would handle before handling the result of saving a card? I believe the app will actually wait for the card to be saved to present a feedback to the user?

I mean, by forking that saga, the user could save another card and while he's doing that he could receive an errore be cose the first card he was trying to save has failed.

**Pull request merged**